### PR TITLE
3 panels in featured content

### PR DESCRIPTION
**Pull request by bethsheets**
Four looked off in dev display, so I removed the older intro video to have 3 panels only in featured content.

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]

File: feat-content.html [hunk 1]
```diff
@@ -20,12 +20,4 @@ <h4 class="feat-content-title">Cloud-optimized WDL Analysis Research Pipelines</
             <h4 class="feat-content-title">FAIR Workflows on Dockstore</h4>
             <p class="feat-card-description">A webinar with tips for how to build a FAIR workflow on Dockstore.</p>
         </a>
-    </div>
```
Comment (coverbeck) [inline feat-content.html:23]: You want to keep this div

**Review by coverbeck — CHANGES_REQUESTED** [review]

File: feat-content.html (hunk 1 above)
Comment (bethsheets) [reply feat-content.html:23]: thanks!

**Review by coverbeck — APPROVED** [review]

**Comment by svonworl** [comment]
The "content-container" div at line 9 in `feat-content.html` might be spurious, it looks wrong and I can't find a matching `</div>`: https://github.com/dockstore/extra-content/blob/fcf0c67df82251bec4af6c37ecabcd94a22d4bc1/feat-content.html#L9

**Review by svonworl — COMMENTED** [review]
See other comment.

**Review by svonworl — APPROVED** [review]

**Pull request merged**